distro_data.json

```diff
@@ -518,18 +518,16 @@
             "eol": "2017-05"
         },
         "versions": {
-            "kernel": "4.1",
-            "GCC": [
-                "4.8",
-                "5.2"
-            ],
-            "clang": "3.7",
-            "glib": "2.44",
+            "pixman": "0.32.6",
+            "glib": "2.44.1",
             "glibc": "2.19",
-            "gtk+2": "2.24",
-            "gtk+3": "3.16",
-            "pixman": "0.32",
-            "stdc++": "5.2"
+            "GCC": [ "4.8", "5.2.1" ],
+            "stdc++": [ "4.8", "5.2.1" ],
```

Comment:
Which one is /usr/lib64/libstdc++.so.6? We only need to know this.

Reply:
Well, that depends on which gcc-version one installs, no? 
For instance: 
```
$ rpm -qli libstdc++6-gcc5.rpm | grep -e Version -e libstd
Name        : libstdc++6-gcc5
Version     : 5.3.1+r233831
/usr/lib64/libstdc++.so.6
/usr/lib64/libstdc++.so.6.0.21
$ rpm -qli libstdc++6-gcc7.rpm | grep -e Version -e libstd
Name        : libstdc++6-gcc7
Version     : 7.4.0+r266845
/usr/lib64/libstdc++.so.6
/usr/lib64/libstdc++.so.6.0.24
```

Reply:
Which one would random users have installed for C++ applications?

Reply:
Ok, there is of course a gcc/libstdc++ version installed by default. But as a package maintainer, I can simply require a newer one, if the application needs that. And it will be automatically installed alongside the application. 
So unless you are interested in doing things on the system by bypassing the package manager, the answer is: "all of them are available".

Reply:
The question for libstdc++ is to know what's available for users that would download Firefox from Mozilla's website rather than install it from a package manager.

Reply:
Ah, I see! Didn't think about that. 
Ok, so I'll change it to the default version.
Sorry, sometimes it takes me a while :-)

Reply:
My latest commits should fix this